packages/lesswrong/components/ultraFeed/UltraFeedItemFooter.tsx

```diff
@@ -258,27 +264,29 @@ const UltraFeedItemFooterCore = ({
         </>
       )}
 
-      {voteProps.document && (
-        <div className={classes.addReactionButton}>
-          <div className={classes.reactionIcon}>
-            <AddReactionButton voteProps={voteProps} />
+      <div className={classes.rightItems}>
```

Comment:
@Will-Howard Just in case you're ever looking the code again, this "rightItems" grouping was breaking. It's ok on desktop but on mobile items are supposed to be spaced evenly

Reply:
Oops sorry, is this fixed elsewhere now or do you want me to fix it?